operations-stdlib/src/commonMain/kotlin/Length.kt

```diff
@@ -0,0 +1,12 @@
+import utils.asList
+import operation.StandardLogicOperation
+
+object Length : StandardLogicOperation {
+    override fun evaluateLogic(expression: Any?, data: Any?): Any? =
+        if (expression.asList.size > 1) null
+        else {
+            val element = expression.asList.firstOrNull()
+            if (element is String) element.toString().length
```

Comment:
might be shortened: (element as? String)?.length

Reply:
also using `with` fits here to avoid casting expression two times
`
    override fun evaluateLogic(expression: Any?, data: Any?): Any? = with(expression.asList) {
        if(size <= 1) {
            (firstOrNull() as? String)?.length
        } else null
    }
`

Reply:
> Did you consider ignoring other values(size >= 2) and using only the first argument?

yap, but that means 
this:
            TestInput(
                expression = mapOf("length" to listOf("test", "test2")),
                result = Failure.NullResult
            ),

returns '4' instead of null result. So that way I'm blocking incorrect usage.

Reply:
all clear 👍🏽